crates/prelude/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 0.5.2-git
+
+### Major
+
+- Change `button::Listener::new()` to never panic and return a `Result<(), Error>` instead.
+
 ## 0.5.1-git
```

Comment:
Sorry, I wasn't clear. What I meant by "bump the version" was going from a patch/minor bump (0.5.1-git) to a major bump (0.6.0-git). See the [Cargo Book](https://doc.rust-lang.org/cargo/reference/resolver.html#semver-compatibility) and the [SemVer](https://semver.org/) specification. A changelog files is a sequence of versions, where a version is a level-2 header with a semver. Only the very first version may have a pre-release version (and when it does, it must be -git). Versions are sorted by precedence (highest precedence first). Each version may have a major, minor, and/or patch level-3 header describing the changes of that category for this version against the previous one. The version number must match the highest level-3 header (major version if there's a major section, minor if minor, patch if patch) including the cargo rule that for 0.x.y, x is major and y minor/patch (and for 0.0.x, x is bumped for all changes, but wasefire doesn't use this numbering scheme). I'll probably have to move this information to some `docs/changelog.md` file.

```suggestion
## 0.6.0-git

### Major

- Change `button::Listener::new()` to never panic and return a `Result<(), Error>` instead.
```

Reply:
Thanks! I wasn't too familiar with SemVer and the Cargo book thank you for the references :+1: